guides/source/active_storage_overview.md

```diff
@@ -490,7 +490,25 @@ directly from the client to the cloud.
     ```erb
     <%= form.file_field :attachments, multiple: true, direct_upload: true %>
     ```
-3. That's it! Uploads begin upon form submission.
+3. Enable Cross-Origin Resource Sharing (CORS): You need to allow direct upload from browser to your cloud storage by enabling CORS. Without this, It will not pass the browser's preflight check. you will see HTTP 403 error when trying to upload. Below are instructions for Amazon S3. if you do not use S3, then please look at the provider specified documentation on how to enable CORS.
```

Comment:
There are some grammar errors in this sentence, but I think overall it would read better like this:

Instead of:

`Without this, It will not pass the browser's preflight check. you will see HTTP 403 error when trying to upload.`

Like this:

`Without this setting you will see HTTP 403 error when trying to upload files.`